repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/openapivalidation/validation/system-model.cfg.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
+    <nodes>
+        <node id="node1" hostname="localhost" http-port="${reposePort}"/>
+    </nodes>
+
+    <filters>
+        <filter name="openapi-validator" uri-regex="/v2/json(/.*)?" configuration="openapi-validator-v2-json.cfg.xml"/>
```

Comment:
Why have we not done this before now? What a great way to avoid restarting repose a million times.

Reply:
I think we have done it before, just sparingly? I'm pretty certain that I have, at least.

I do like this better than restarting a million times, but there are a couple of downsides:

1. If you are testing an invalid configuration, this doesn't really work since a single invalid configuration will cause Repose to return 503s. So invalid configuration tests still need to be isolated.
1. This approach does complicate configuration a bit since you have to match requests not just to a set of configs, but to an execution path within a larger set of configs.